app/components/UserLibraryList.vue

```diff
@@ -1,81 +1,107 @@
 <script setup lang="ts">
-  const annoteDocs = ref<AnnoteDocument[] | null>(null);
-  const { getCurrentUser } = useAuth();
-  
-  const currentUser = await getCurrentUser();
- 
-  onMounted(async () => {
-    const { data: fetchedDocument } = await $fetch<ApiResponse<AnnoteDocument[]>>('/api/annote_documents');
-    if (fetchedDocument) {
-      annoteDocs.value = fetchedDocument;
-    }
-   
-  });
+const annoteDocs = ref<AnnoteDocument[] | null>(null);
+const { getCurrentUser } = useAuth();
+
+const currentUser = await getCurrentUser();
+
+onMounted(async () => {
+  const { data: fetchedDocument } = await $fetch<ApiResponse<AnnoteDocument[]>>(
+    "/api/annote_documents"
+  );
+  if (fetchedDocument) {
+    annoteDocs.value = fetchedDocument;
+  }
+});
 </script>
 
 <template>
-  <div class="mt-4">
+  <div class="w-2/3 mx-auto mt-4">
+    <h1 class="text-2xl">Library</h1>
     <!-- This search bar section -->
-    <div class="py-4 flex justify-end gap-x-2 pr-4">
-
-      <div class="searchTextField lightRoundedGreyBorder flex">
-        <div class="self-center mt-2">
+    <div class="flex justify-end py-4 pr-4 gap-x-2">
+      <div class="flex searchTextField lightRoundedGreyBorder">
+        <div class="self-center mt-2 ml-2">
           <Icon name="mdi:magnify" color="black" size="1.5rem" />
         </div>
-        <input type="text" placeholder="Search" class="searchInput w-full p-2 border border-black" />
+        <input
+          type="text"
+          placeholder="Search"
+          class="w-full p-2 border border-black searchInput"
+        />
```

Comment:
For the search and the dropdown, on smaller sizes, I'm thinking they should appear full-width in a column?

Reply:
Sounds good to me